File: python/masrobo_rcs_sdk/robot_controller.py
from .client import Client
from .config import Config
from .iot_device import IotDeviceService

class RobotController:
    def __init__(self, base_url: str, app_id: str, app_key: str, session=None):
        client = Client(Config(base_url=base_url, app_id=app_id, app_key=app_key, session=session))
        self.IotDevice = IotDeviceService(client)

File: java/src/main/java/com/boticz/masrobo/RobotController.java
package com.boticz.masrobo;

import com.boticz.masrobo.service.IotDeviceService;
import com.boticz.masrobo.client.Config;
import com.boticz.masrobo.client.OpenHttpClient;


public class RobotController {
    private final IotDeviceService iotDevice;

    public RobotController(Config config) {
        OpenHttpClient httpClient = new OpenHttpClient(config);
        this.iotDevice = new IotDeviceService(httpClient);
    }

    public IotDeviceService getIotDevice() {
        return iotDevice;
    }
}

